protostar/commands/test/test_results.py

```diff
@@ -47,6 +47,11 @@ class BrokenTestCaseResult(TimedTestCaseResult):
     exception: ReportedException
 
 
+@dataclass(frozen=True)
+class SkippedTestCaseResult(TimedTestCaseResult):
+    reason: Optional[str]
```

Comment:
not applicable now?

Reply:
still used for mapping SkippedSetupCaseResult to SkippedTestCaseResult